internal/controller/validator.go

```diff
@@ -16,6 +17,8 @@ import (
 	"github.com/openshift/oadp-operator/pkg/credentials"
 )
 
+var wasRestic bool
```

Comment:
Should we explicitly set this as false ? (I think default is false but still)

Reply:
If we do, it would fail https://revive.run/r#var-declaration linter that was recently merged https://github.com/vmware-tanzu/velero/pull/8636/files#diff-9917ddc9f1c3304218f7269265b746d997c5c0615478177b5fceecd33ef47cb5
specifically because we are [declaring "zeroLiteral"](https://github.com/mgechev/revive/blob/d9c61c1518f16418ad94d92b2d5976c266e0ad6e/rule/var_declarations.go#L13-L26)
```go
var zeroLiteral = map[string]bool{
	"false": true, // bool
	// runes
	`'\x00'`: true,
	`'\000'`: true,
	// strings
	`""`: true,
	"``": true,
	// numerics
	"0":   true,
	"0.":  true,
	"0.0": true,
	"0i":  true,
}
```